src/components/Header/Pages.js

```diff
@@ -35,6 +38,7 @@ const playerPages = [{
   name: 'Counts',
 }, {
   name: 'Histograms',
+  extra: histogramNames[0],
```

Comment:
I could do that, but I will still need all this code because otherwise we can't highlight the tab in the tabbar if they click on any of the buttons. The only way to highlight the tab is to check the value, so if they click on any other histogram or route to a histogram directly, the tab in the player bar won't show as active on histograms. I think it's ok for this to be like this as I can see lots of scenarios where tabs will have routing inside them but we still want that tab to show up as highlighted.